src/mutability/Mutator.sol

```diff
@@ -48,13 +66,19 @@ contract Mutator is IMutator, Derived, Pausable {
         _nameToMutable[name].upgrade{value: msg.value}(implementation, data);
     }
 
-    function _pause() internal override {
+    function _pause() internal {
         for (uint256 i = 0; i < _mutables.length(); i++) IMutable(_mutables.at(i)).pause();
-        super._pause();
+        emit IMutableTransparent.Paused();
```

Comment:
I think we can forgo this event, the `IMutableTransparent`'s should be emmitting their own version of these

Reply:
Indeed; removed in 227e58d29400c44cd310fbad52198717f386a562.